# Build wow_helper instead of using the one included with Chromium

--- a/sandbox/win/sandbox_win.gypi
+++ b/sandbox/win/sandbox_win.gypi
@@ -184,16 +184,10 @@
       'include_dirs': [
         '../..',
       ],
-      'target_conditions': [
+      'conditions': [
         ['target_arch=="ia32"', {
-          'copies': [
-            {
-              'destination': '<(PRODUCT_DIR)',
-              'files': [
-                'wow_helper/wow_helper.exe',
-                'wow_helper/wow_helper.pdb',
-              ],
-            },
+          'dependencies': [
+            'win/wow_helper/wow_helper.gyp:wow_helper',
           ],
         }],
       ],
--- /dev/null
+++ b/sandbox/win/wow_helper/wow_helper.gyp
@@ -0,0 +1,26 @@
+{
+  'targets': [
+    {
+      'target_name': 'wow_helper',
+      'type': 'executable',
+      'configurations': {
+        'Common_Base': {
+          'msvs_target_platform': 'x64',
+          'msvs_settings': {
+            'VCLinkerTool': {
+              'SubSystem': '2',
+            },
+          },
+        },
+      },
+      'include_dirs': [
+        '../../..',
+      ],
+      'sources': [
+        'service64_resolver.cc',
+        'target_code.cc',
+        'wow_helper.cc',
+      ],
+    },
+  ],
+}
